staging/istio/charts/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 description: A Helm chart for Kubernetes
 name: grafana
-version: 1.4.3
-appVersion: 1.4.3
+version: 1.1.0
+appVersion: 1.1.0
```

Comment:
Did you expect this to be downgraded?

Reply:
Thats how it is in upstream. I guess because they moved to istio-operator maybe they started with different version